homeassistant/helpers/entity_platform.py

```diff
@@ -426,11 +426,13 @@ async def setup_again(*_args: Any) -> None:
                 type(exc).__name__,
             )
             return False
-        except Exception:
+        except Exception as exc:
+            message = str(exc)
```

Comment:
We don't need a local variable. The exception has a string magic method so we don't need to copy to string explicitly first. Just pass the exception as a log variable.

Reply:
Hi @MartinHjelmare, thanks for reviewing it so quickly.
When I do what you suggest I get the following Ruff warning:
Redundant exception object included in `logging.exception` call Ruff[TRY401](https://docs.astral.sh/ruff/rules/verbose-log-message).
Not sure why I get this warning as there is clear value with adding the additional %s to the final message as without it the only way to see the actual error is by clicking the error and see it in the popup.

Reply:
Disable that ruff rule for this line then.

Reply:
Disabled.